tests/utils_test.py

```diff
@@ -185,25 +185,19 @@ def test_ulimit_invalid_type(self):
         self.assertRaises(ValueError, lambda: Ulimit(name='hello', hard='456'))
 
     def test_create_host_config_dict_logconfig(self):
-        dct = {'type': LogConfig.types.SYSLOG, 'config': {'key1': 'val1'}}
+        dct = {'type': 'syslog', 'config': {'key1': 'val1'}}
         config = create_host_config(log_config=dct)
         self.assertIn('LogConfig', config)
         self.assertTrue(isinstance(config['LogConfig'], LogConfig))
         self.assertEqual(dct['type'], config['LogConfig'].type)
 
     def test_create_host_config_obj_logconfig(self):
-        obj = LogConfig(type=LogConfig.types.SYSLOG, config={'key1': 'val1'})
+        obj = LogConfig(type='syslog', config={'key1': 'val1'})
         config = create_host_config(log_config=obj)
         self.assertIn('LogConfig', config)
         self.assertTrue(isinstance(config['LogConfig'], LogConfig))
         self.assertEqual(obj, config['LogConfig'])
 
-    def test_logconfig_invalid_type(self):
-        self.assertRaises(ValueError, lambda: LogConfig(type='xxx', config={}))
-        self.assertRaises(ValueError, lambda: LogConfig(
```

Comment:
The validation code that would have raised a ValueError has been removed, thus making this test redundant. The tests added in integration_tests cover the use case of invalid config types and check that we receive an error back from the API.